yb-voyager/src/query/queryparser/helpers_struct.go

```diff
@@ -264,3 +268,69 @@ func IsDDL(parseTree *pg_query.ParseResult) (bool, error) {
 	//Not Full-proof as we don't have all DDL types but atleast we will skip all the types we know currently
 	return !ok, nil
 }
+
+func IsJsonbType(node *pg_query.Node, jsonbColumns []string, jsonbFunctions []string) bool {
```

Comment:
how about `DoesNodeEvaluateToJsonB` or something similar? add some doc to this function. Explain that we are checking if the particular node, when evaluated, results in a jsonb value.. could be a column, a typecase, a function call, an expression, etc,etc.